### ISSUE TITLE
AutoSub not downloading anymore

### ISSUE THREAD

**Issue by TechnoRage** [BenjV/autosub#51]
AutoSub not downloading anymore:

05-04 08:10:20 INF A7_Login       :Could not login with username: TechnoRage
05-04 08:10:20 ERR getA7Info      :Couldn't retrieve Addic7ed account info for TechnoRage. Maybe not logged in.
05-04 08:10:18 INF checkSub       :Starting round of subs searching on Addic7ed,Opensubtitles,Subscene,Podnapisi

**Comment by TechnoRage** [comment]
Sorry, moest net weg, vandaar geen aanvullende info.

Maar AutoSub versie 1.2.23 download een tijdje al niks meer en als ik de settings wil aanpassen, zegt 'ie dat er een zoekopdracht gaande is en de instellingen niet kan opslaan. Ook de testknoppen voor Addic7ed en Open subtitles zijn grijs, dus deze kan ik niet gebruiken.

Please help :S

**Comment by BenjV** [comment]
Addic7ed heeft een captcha ingevoerd dus die site kan even niet gebruikt worden.
Waarschijnlijk loopt hij daar bij jou ook op vast.
Autosub even stoppen en addic7ed op "False" zetten in de config.properties.
Als je zelf de config.properties niet kunt aanpassen, kun je autosub stoppen, de seriefolder een andere naam geven autosub starten en dan op de configuration page addic7ed uitzetten.
Daarna kun je de serie folder weer renamen naar de oude naam en kan autosub weer verder met alleen opensubtitles.

Ik ben met het addic probleem bezig en er komt een oplossing binnenkort.

**Comment by TechnoRage** [comment]
Ha BenjV, bedankt voor je reactie!
Ik heb, zij het met wat moeite, de instellingen kunnen aanpassen dmv het hernoemen van de subs-directory. Echter blijft het e.e.a. nog niet lekker werken; m'n home screen blijft op 'Search is busy...' staan:
![image](https://user-images.githubusercontent.com/26321025/55777750-2942f800-5aa1-11e9-8340-d38723b550bc.png)

Btw, waar staat het config.properties bestand?:
![image](https://user-images.githubusercontent.com/26321025/55777851-7aeb8280-5aa1-11e9-9054-6807b1cca795.png)

**Comment by TechnoRage** [comment]
Ik heb hem al gevonden:

![image](https://user-images.githubusercontent.com/26321025/55778635-aec7a780-5aa3-11e9-8867-9ae655a48d48.png)

Kan het misschien helpen het pakket volledig te verwijderen en opnieuw te installeren? (heb je een laatste versie?)

**Comment by BenjV** [comment]
De laatste beschikbre versie is 1.2.23, ik ben met een nieuwe versie beig.
De plaatjes die je laat zien heb ik niet veel aan, beter is het om het laatste stuk van de inhoud van AutoSubServices.log te laten zien.

Wat er verder aan de hand kan zijn is dat je podnapisi en/of subscene nog aan hebt staan, want die werken niet meer omdat de website subtitleseeker verdween is.
Ook kun je een serie hebben waar geen addic mapping voor is en dan gaat hij alsnog op addic7ed zoeken wat niet lukt door de captcha en dus continue op timeouts van addic7ed zit te wachten.

Al deze problemen worden in de komende release opgelost.

**Comment by TechnoRage** [comment]
Dat was het; ik heb podnapsi en subscene uitgezet:
![image](https://user-images.githubusercontent.com/26321025/55785324-84c9b180-5ab2-11e9-9b15-4f9b40ab738c.png)

(ander vraagje: wat zijn jouw beste ervaringen vwb "MinMatchScore" en "Must match groups"?)

Avast bedankt weer!

**Comment by RabadersGit** [comment]
Any progress finding a solution for Addic7ed? Currently only OpenSubtitles work.

**Comment by BenjV** [comment]
Yes there is lots of progress, but also still lots of work to do.
I will try to get a new version before the new show season begins.

**Comment by marcofranssen** [comment]
Maybe related to #53. For me it works again.
